scater-calculate-qc-metrics.R

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env Rscript 
+
+# Load optparse we need to check inputs
+suppressPackageStartupMessages(require(optparse))
+
+# Load common functions
+suppressPackageStartupMessages(require(workflowscriptscommon))
+
+# parse options
+option_list = list(
+  make_option(
+    c("-i", "--input-object-file"),
+    action = "store",
+    default = NA,
+    type = 'character',
+    help = "File name in which a serialized R SingleCellExperiment object where object matrix found"
```

Comment:
Can I be nit-picky and request de-capitalisation here? Just for consistency with other help text.